blocks/serialization.py

```diff
@@ -215,10 +218,13 @@ def secure_dump(object_, path, dump_function=dump, **kwargs):
 
     """
     try:
+        logger.debug("Dump object to a temporaty file")
```

Comment:
http://dictionary.cambridge.org/dictionary/english/dump - an ​act of ​moving ​information from a computer's ​memory to another ​place or ​device. I was also not sure, but the dictionary says it's ok.